### PR TITLE
[Sysdig] Add Node Analyzer

### PR DESCRIPTION
**Pull request by noahkraemer**
This PR adds the Node Analyzer daemonset configmaps and helm parameters.

## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)
- [x] PR only contains changes for one chart

### CODE REVIEW

File: charts/sysdig/values.yaml [hunk 1]
```diff
@@ -247,6 +247,117 @@ nodeImageAnalyzer:
...
+      requests:
+        cpu: 150m
+        memory: 512Mi
+      limits:
+        cpu: 500m
+        memory: 1536Mi
+  hostAnalyzer:
+    image:
+      repository: sysdig/host-analyzer
+      tag: 0.1.10 # TODO pinned or latest?
```
Comment (noahkraemer) [inline charts/sysdig/values.yaml]: What should this tag be?

File: charts/sysdig/README.md [hunk 2]
```diff
@@ -43,70 +43,107 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `nodeAnalyzer.imageAnalyzer.resources.requests.memory`     | Node Image Analyzer Memory requests per node                                             | `512Mi`                                                                       |
+| `nodeAnalyzer.imageAnalyzer.resources.limits.cpu`          | Node Image Analyzer CPU limit per node                                                   | `500m`                                                                        |
+| `nodeAnalyzer.imageAnalyzer.resources.limits.memory`       | Node Image Analyzer Memory limit per node                                                | `1536Mi`                                                                      |
+| `nodeAnalyzer.hostAnalyzer.image.repository`               | The image repository to pull the Host Analyzer from                                      | `sysdig/host-analyzer`                                                        |
+| `nodeAnalyzer.hostAnalyzer.image.tag`                      | The image tag to pull the Host Analyzer                                                  | `TODO 0.1.7`                                                                  |
```
Comment (noahkraemer) [inline charts/sysdig/README.md]: What should this tag be?

File: charts/sysdig/README.md [hunk 3]
```diff
@@ -125,7 +162,7 @@ $ helm install --namespace sysdig-agent sysdig-agent -f values.yaml sysdig/sysdi
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
 ## Node Image Analyzer
-
+ TODO Update
```
Comment (noahkraemer) [inline charts/sysdig/README.md]: We will still default to installing the NIA. Should we change the default to the NA? Should we remove the NIA entirely?

File: charts/sysdig/values.yaml (hunk 1 above)
Comment (nestorsalceda) [reply charts/sysdig/values.yaml]: We use pinned versions following the convention: 

https://helm.sh/docs/chart_best_practices/pods/#images

Although it involves a bit of more work in our side, I would keep doing it.

File: charts/sysdig/README.md (hunk 2 above)
Comment (nestorsalceda) [reply charts/sysdig/README.md]: Same than in the previous

File: charts/sysdig/README.md (hunk 3 above)
Comment (nestorsalceda) [reply charts/sysdig/README.md]: I think we should promote the NA, perhaps we can add a major version bump to make this clear.

File: charts/sysdig/values.yaml (hunk 1 above)
Comment (noahkraemer) [reply charts/sysdig/values.yaml]: Agreed =) I just need the latest version so I can use that

File: charts/sysdig/README.md (hunk 3 above)
Comment (noahkraemer) [reply charts/sysdig/README.md]: Sounds good to me 👍

File: charts/sysdig/values.yaml (hunk 1 above)
Comment (nestorsalceda) [reply charts/sysdig/values.yaml]: The CI failed, I think is because the comment

Once you remove it, it will be fine

**Comment by nestorsalceda** [comment]
Thank you so much @nkraemer-sysdig !!

**Pull request merged**